samples/csharp/roslyn-sdk/SyntaxQuickStart/HelloSyntaxTree/Program.cs

```diff
@@ -0,0 +1,87 @@
+using static System.Console;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace HelloSyntaxTree
+{
+    class Program
+    {
+
+        // <Snippet1>
+        const string programText =
+@"using System;
+using System.Collections;
+using System.Linq;
+using System.Text;
+ 
+namespace HelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(""Hello, World!"");
+        }
+    }
+}";
+        // </Snippet1>
+
+        static void Main(string[] args)
+        {
+            // <Snippet2>
+            SyntaxTree tree = CSharpSyntaxTree.ParseText(programText);
+            var root = tree.GetRoot() as CompilationUnitSyntax;
+            // </Snippet2>
+
+            // <Snippet3>
+            WriteLine($"The tree is a {root.Kind().ToString()} node.");
+            WriteLine($"The tree has {root.Members.Count} elements in it.");
+            WriteLine($"The tree has {root.Usings.Count} using statements. They are:");
+            foreach (UsingDirectiveSyntax element in root.Usings)
+                WriteLine($"\t{element.Name.ToString()}");
+            // </Snippet3>
+
+            // <Snippet4>
+            MemberDeclarationSyntax firstMember = root.Members[0];
+            WriteLine($"The first member is a {firstMember.Kind()}");
+            NamespaceDeclarationSyntax helloWorldDeclaration = firstMember as NamespaceDeclarationSyntax;
+            // </Snippet4>
+
+            // <Snippet5>
+            WriteLine($"There are {helloWorldDeclaration.Members.Count} members declared in this namespace.");
```

Comment:
should you account for 1 vs 1+ here? On your results, you ended up with the following sentence:
"There are 1 members declared in this namespace."

Not sure if you want "There is 1 member declared in this namespace." and "There are n members declared in this namespace." for n=2 or more.

Reply:
went with singular.